packages/react-ui-core/src/Gmap/Markers.js

```diff
@@ -0,0 +1,69 @@
+import { Component } from 'react'
+import PropTypes from 'prop-types'
+import { setupMarker, removeMarker } from './utils/markerHelpers'
+
+const MARKER = ({ properties, geometry }) => ({
+  id: properties.id,
+  position: {
+    lat: geometry.coordinates[1],
```

Comment:
is this order guaranteed? also seems a bit weird, as`lat` always come before `lng` normally

Reply:
Yeah I dunno, I should double check the geojson standard. I took this directly from ag.js so who knows if it's right lol.

Reply:
looks like Mapbox and GeoJSON prefer `long, lat`, but Google Maps API prefers `lat, long`, so, yayyyy

Reply:
The good thing is you can still override this if you wanted 😄

Reply:
From the [RFC](https://tools.ietf.org/html/rfc7946)
```
9.  Mapping 'geo' URIs

   'geo' URIs [RFC5870] identify geographic locations and precise (not
   uncertain) locations can be mapped to GeoJSON Geometry objects.

   For this section, as in [RFC5870], "lat", "lon", "alt", and "unc" are
   placeholders for 'geo' URI latitude, longitude, altitude, and
   uncertainty values, respectively.

   A 'geo' URI with two coordinates and an uncertainty ('u') parameter
   that is absent or zero, and a GeoJSON Point geometry may be mapped to
   each other.  A GeoJSON Point is always converted to a 'geo' URI that
   has no uncertainty parameter.

   'geo' URI:

   geo:lat,lon

   GeoJSON:

   {"type": "Point", "coordinates": [lon, lat]}

   The mapping between 'geo' URIs and GeoJSON Points that specify
   elevation is shown below.

   'geo' URI:

   geo:lat,lon,alt

   GeoJSON:

   {"type": "Point", "coordinates": [lon, lat, alt]}

   GeoJSON has no concept of uncertainty; imprecise or uncertain 'geo'
   URIs thus cannot be mapped to GeoJSON geometries.
```

Reply:
Seriously? So they don't even follow the geojson standard?
![](https://media3.giphy.com/media/AjYsTtVxEEBPO/giphy.gif)